sdk/core/core-tracing/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/core-tracing",
-  "version": "1.0.0-preview.7",
+  "version": "1.0.0-preview.8",
```

Comment:
Are we bumping the version of core-tracing used in storage, keyvault, event hubs etc in a separate PR or we can do it here?

Reply:
I am updating the rest libs in a separate PR

Reply:
including them in this PR instead.